### PR TITLE
10999

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#10999

*Description of changes:*

- stored account id in session
- created "navigate" function to automatically add accountid parameter
- replaced window.location.href with new function
- fixed date/time conversions broken by Node 12
- fixed missing prop error
- app version = 3.0.9

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: src/services/namex.services.ts [hunk 1]
```diff
@@ -238,8 +238,8 @@ export default class NamexServices {
       const blob = new Blob([response.data], { type: 'application/pdf' })
 
       // IE doesn't allow using a blob object directly as link href, so use msSaveOrOpenBlob
-      if (window.navigator && window.navigator.msSaveOrOpenBlob) {
-        window.navigator.msSaveOrOpenBlob(blob)
+      if (window.navigator && window.navigator['msSaveOrOpenBlob']) {
+        window.navigator['msSaveOrOpenBlob'](blob, 'Name Request Results')
```
Comment (severinbeauvais) [inline src/services/namex.services.ts:242]: I did this (as in our other UIs) to bypass the warning, "property 'msSaveOrOpenBlob' does not exist on type 'Navigator'."

File: package-lock.json [hunk 2]
```diff
@@ -5215,9 +5215,9 @@
       }
     },
     "caniuse-lite": {
-      "version": "1.0.30001230",
-      "resolved": "https://registry.npmjs.org/caniuse-lite/-/caniuse-lite-1.0.30001230.tgz",
-      "integrity": "sha512-5yBd5nWCBS+jWKTcHOzXwo5xzcj4ePE/yjtkZyUV1BTUmrBaA9MRGC+e7mxnqXSA90CmCA8L3eKLaSUkt099IQ==",
+      "version": "1.0.30001309",
```
Comment (severinbeauvais) [inline package-lock.json:5218]: This needs to be updated regularly per NPM message:
```
Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db
```

**Review by eve-git — APPROVED** [review]
LGTM

**Review by cameron-freshworks — APPROVED** [review]
Looks great!

File: src/services/namex.services.ts (hunk 1 above)
Comment (severinbeauvais) [reply src/services/namex.services.ts:242]: We can _probably_ remove this IE code in all UIs. (Next time we're in this area of code.)

Just the fact that the function no longer exists on `window.navigator` tells me IE support is deprecated/obsolete.

**Review by ozamani9 — APPROVED** [review]
Looks good

**Pull request merged**